src/services/auth/plugins/password/service.ts

```diff
@@ -40,6 +66,104 @@ export class MemberPasswordService {
     await memberPasswordRepository.patch(actor.id, newPassword);
   }
 
+  /**
+   * Modify the password of a member. Force the change without checking the current password.
+   * Check if the token is registered in the redis database. If it is, delete it and change the password of the associated member.
+   * @param repositories Object with the repositories needed to interact with the database. Must contain a memberPasswordRepository.
+   * @param password New password.
+   * @param token The JSON Web Token associated to the member that wants to reset the password.
+   * @returns void
+   */
+  async forcePatch(repositories: Repositories, password: string, token: string): Promise<void> {
```

Comment:
Does this need to be called `forcePatch`? Could this be `reset`?  I understand the need for `force`, but it does not explain really what it does.